src/main/java/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel.java

```diff
@@ -72,9 +74,19 @@ public static FlowDurabilityHint getSuggestedDurabilityHint() {
         }
 
         @Nonnull
-        @Override
         public Permission getRequiredGlobalConfigPagePermission() {
-            return Jenkins.MANAGE;
+            return getJenkinsManageOrAdmin();
+        }
+
+        // TODO: remove when Jenkins core baseline is 2.222+
+        Permission getJenkinsManageOrAdmin() {
+            Permission systemRead;
+            try { // System Read is available starting from Jenkins 2.222 (https://jenkins.io/changelog/#v2.222). See JEP-224 for more info
+                systemRead = (Permission) ReflectionUtils.getPublicProperty(Jenkins.get(), "SYSTEM_READ");
```

Comment:
Shoudn't this be "MANAGE" instead of "SYSTEM_READ"?

Reply:
:facepalm: you are totally right. Fixed.

Reply:
Good catch BTW! Thanks.